src/Tests/AuthoringTest/Program.cs

```diff
@@ -1575,6 +1575,35 @@ public partial struct PartialStruct
         public double Z;
     }
 
+    // Nested type to validate (https://github.com/microsoft/CsWinRT/issues/1477)
+    // Doesn't need to be consumed, we just want to verify the generator does work.
+    internal partial class Nested1

```

Comment:
Not sure why this got included here again, probably due to the force push in the other PR.